src/skottie-player.ts

```diff
@@ -489,6 +492,46 @@ export class SkottiePlayer extends LitElement {
     }
   }
 
+  private bodySegment(reader: FileReader, file: File, mapKey: string): void {
+    const url = URL.createObjectURL(file);
+    const oldCanvas = document.getElementById('canvas');
+    if (oldCanvas) {
+      oldCanvas.remove();
+    }
+    const img = new Image();
+    img.onload = () => { 
+      const canvasElement = document.createElement('canvas');
+      canvasElement.setAttribute('id', 'canvas');
+      canvasElement.setAttribute('width', img.width.toString());
+      canvasElement.setAttribute('height', img.height.toString());
+      const context = canvasElement.getContext('2d');
+      if (context) {
+        context.drawImage(img, 0, 0, 
+          img.width, img.height);
+        let myData = context.getImageData(0, 0,
+          img.width, img.height);
+        console.log('segmenting');
+        this.bodySegmenter.segmentPerson(myData).then((result) => {
+          console.log('segmented');
+          for (let i = 0; i < result.length; i++) {
+            if (result[i] === 0) {
+              myData.data[i * 4 + 3] = 0;
+            }
```

Comment:
A bit unclear about the logic behind the if block above.... Consider adding some comments.